CHANGELOG.md

```diff
@@ -4,90 +4,107 @@
 
 - Added new beta `RateService`, accessible via `myClient.Beta.Rate`.
 - Added `RetrieveStatelessRate` function under beta `RateService` to pull stateless rates when shipment data is provided
-- Adds `GetLowestStatelessRate` function under `RateService` and under `Calculations.Rate` to filter the lowest stateless rate
-- Adds new `GetLowest` instance functions, callable on `List<Rate>`, `List<Smartrate>` and `List<StatelessRate>`, to filter the lowest rate
+- Added `GetLowestStatelessRate` function under `Utilities.Rate` to filter the lowest stateless rate
+- Added new `GetLowest` instance functions, callable on `List<Rate>`, `List<Smartrate>` and `List<StatelessRate>`, to
+  filter the lowest rate
+- Deprecated rate and smartrate filtering methods in `ShipmentService`, `RateService` and `Calculation.Rates` namespaces, moved to `Utilities.Rate` namespace
 
 ## v4.3.0 (2023-01-18)
 
-- Added payload functions `RetrieveAllPayloadsForEvent` and `RetrievePayloadForEvent` methods, accessible via `myClient.Event` service.
+- Added payload functions `RetrieveAllPayloadsForEvent` and `RetrievePayloadForEvent` methods, accessible
+  via `myClient.Event` service.
 - Added function to retrieve all pickups via `myClient.Pickup.All()`
 
 ## v4.2.0 (2023-01-11)
 
 - Added new beta billing functionality for referral customer users, accessible via `myClient.Beta.Referral` service
-  - `AddPaymentMethod` to add an existing Stripe bank account or credit card to your EasyPost account
-  - `RefundByAmount` refunds you wallet balance by a specified amount
-  - `RefundByPaymentLog` refunds you wallet balance by a specified payment log
+    - `AddPaymentMethod` to add an existing Stripe bank account or credit card to your EasyPost account
+    - `RefundByAmount` refunds you wallet balance by a specified amount
+    - `RefundByPaymentLog` refunds you wallet balance by a specified payment log
```

Comment:
Please put-back all the changes to CHANGELOG, should only be the new entries that got affected